okhttp/src/main/java/com/squareup/okhttp/ConnectionSpec.java

```diff
@@ -259,7 +259,7 @@ public Builder tlsVersions(TlsVersion... tlsVersions) {
       return tlsVersions(strings);
```

Comment:
... and if you do add the defensive copy, you should change this method to avoid self-use because that will save an unnecessary allocation. You can just assign `this.tlsVersions` right here and `return this`.